docs/source/tutorials/base/demo.rst

```diff
@@ -1,6 +1,66 @@
 How to run the demo with exportable code
 ========================================
 
-<Guidelines for writing>
+In this tutorial we will show how to infer the model deployed in the previous step on images, videos, and webcam in order and see how it works on custom data.
 
-Explain what exportable code is. Concrete model. How to use demonstration mode. Concrete example 
\ No newline at end of file
+1. Unzip the ``openvino.zip`` archive.
+
+.. code-block::
+
+    unzip  outputs/deploy/openvino.zip -d outputs/deploy/
+
+2. To run the demo in exportable code, we can use a brand-new virtual environment, where we need to install the minimum packages from exportable code.
```

Comment:
Here could be smthg like: we need to install a minimalistic set of packages required for inference.
It is not clear how to install something from the exportable code itself.